src/EVM/SMT.hs

```diff
@@ -378,7 +411,7 @@ prelude =  (flip SMT2) mempty $ fmap (fromLazyText . T.drop 2) . T.lines $ [i|
   )
 
   ; buffers
-  (declare-fun bufLength (Buf) Word)
+  ; (declare-fun bufLength (Buf) Word)
```

Comment:
I think we can just delete this line entirely?

Reply:
Yes! Done!